### DISCUSSION TITLE
Montage Command

### DISCUSSION THREAD

**Discussion by byumark** [dlemstra/magick-wasm#67] [General]
Is it possible to do the montage command (or similar)?

**Comment by dlemstra** [comment]
Possible yes, but this is not yet available in the API. Will make this the next operation to add.

**Reply by dlemstra** [reply]
It took me a while before making time to work on this but this will become available in the next release.

**Reply by byumark** [reply]
Thank you, @dlemstra !

**Reply by dlemstra** [reply]
A new releases that adds support for montage has just been published.
